src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const myFun = (a) => {
```

Comment:
use correct names for variables

Reply:
agree, myFun isn't a good name for variable